CONTRIBUTING.rst

```diff
@@ -42,17 +51,18 @@ The minimum requirements for a new feature are:
 
 Important Guidelines
 --------------------
- Pylanetary intends to provide *generic* tools for solar system data processing and modeling.
- As such, all functions and classes should remain agnostic to planet, observatory, 
- wavelength band, etc. Defaults for a given planet, observatory, wavelength band, etc.
- may be provided as .yaml (preferred) or another text file format, and should go in the ``pylanetary/feature/data`` subdirectory. See ``pylanetary/rings/data`` for an example.
- 
- Features that are primarily relevant to a single observatory, planet, wavelength band, etc.
- should be placed in separate GitHub repositories and import pylanetary. Again, if portions 
- of that functionality are generic, they can be included in pylanetary 
- 
- Docstrings should be written for every function and should adhere to the same
- format as shown in the functions in ``pylanetary/feature/planetnav/core.py``
+Pylanetary intends to provide *generic* tools for solar system data processing and modeling.
+As such, all functions and classes should remain agnostic to planet, observatory, 
+wavelength band, etc. Defaults for a given planet, observatory, wavelength band, etc.
+may be provided as .yaml (preferred) or another text file format, and should go in the ``pylanetary/feature/data`` subdirectory. See ``pylanetary/rings/data`` for an example.
+
+Features that are primarily relevant to a single observatory, planet, wavelength band, etc.
+should be placed in separate GitHub repositories and import pylanetary. Again, if portions 
+of that functionality are generic, they can be included in pylanetary 
+
+Docstrings should be written for every function and should adhere to the same
```

Comment:
this is complicated to do because there are so many different options and use cases.
Should I show how to write a minimal docstring for a very simple function?  Should I include a docstring like the one for Nav.colocate(), with multiple different options that each have their own kwargs?  Should I show every possible section header and ask people to choose which are useful to them?  Should I show how to include documentation for a function that writes files?  Instead, I chose to just refer people to a page that has many of those use-cases already